packages/DropdownMenu/src/components/Item/Item.js

```diff
@@ -28,27 +31,39 @@ const propTypes = {
 const defaultProps = {
   isDestructive: false,
   onClick: () => {},
+  onKeyDown: () => {},
   onClose: () => {},
   onShowConfirmation: () => {},
   renderConfirmation: null,
 };
 
 const Item = props => {
-  const { children, isDestructive, onClick, onClose, onShowConfirmation, renderConfirmation, ...moreProps } = props;
+  const {
+    children,
+    isDestructive,
+    onClick,
+    onClose,
+    onShowConfirmation,
+    renderConfirmation,
+    onKeyDown,
+    ...moreProps
+  } = props;
 
   const handleClickItem = () => {
-    onClick();
+    onClick(children);
     onClose();
   };
 
   const itemProps = {
     onClick: renderConfirmation !== null ? onShowConfirmation : handleClickItem,
     role: "menuitem",
+    className: "dropdown-item",
+    onKeyDown,
     isDestructive,
   };
 
   return (
-    <RawButton hasInsetFocusStyle css={ItemStyles} {...itemProps} {...moreProps}>
+    <RawButton tabIndex={0} hasInsetFocusStyle css={ItemStyles} {...itemProps} {...moreProps}>
```

Comment:
I am setting the tabIndex specifically as 0 so that even if this item is disabled it can take focus. This has required a small change in how the `RawButton` component evaluates tab index. https://github.com/acl-services/paprika/pull/318/files#diff-57b7f13a30b6174f5a9c7477773ad940R99